### PR TITLE
Change party-robot info to only mention fmt.Sprintf (#1597)

### PR DESCRIPTION
**Pull request by bobtfish**

### CODE REVIEW

**Comment by junedev** [comment]
@ErikSchierboom I would recommend having one sentence in the introduction that explains the difference between Printf and Sprintf as this confuses many students that come from languages that don't have those functions. (You suggested to completely remove Printf in the issue description https://github.com/exercism/go/issues/1597 which is what was done in this PR.)  
[The docs](https://exercism.org/docs/building/tracks/concept-exercises) describe this kind of information as what "is needed to understand the fundamentals of the concept". WDYT?

**Review by junedev — CHANGES_REQUESTED** [review]
As said here https://github.com/exercism/go/pull/1650#issuecomment-922326056 I would strongly recommend explaining the difference between Printf and Sprintf.

File: exercises/concept/party-robot/.docs/introduction.md [hunk 1]
```diff
@@ -8,32 +8,22 @@ A `string` value can be written as a string literal, which is a sequence of byte
...
-
-```go
-n := 12
-s := fmt.Sprintf("%d laps", n)
-fmt.Printf("Do %s!", s)
-// Output: Do 12 laps!
-```
-
-For more details, see the [`fmt` package documentation](https://pkg.go.dev/fmt).
+`fmt` contains other functions for working with strings, such as `Println` which simply prints the input received and `Printf` which formats the input in the same was as `Sprintf` beford print it.
```
Comment (junedev) [inline exercises/concept/party-robot/.docs/introduction.md:29]: The following suggestion contains some fixed typos and a minor
```suggestion
`fmt` contains other functions for working with strings, such as `Println` which simply prints the arguments it receives to the console and `Printf` which formats the input in the same way as `Sprintf` before printing it.
```

**Review by junedev — CHANGES_REQUESTED** [review]
There are some typos in the new sentence you added, see suggestion.

**Comment by bobtfish** [comment]
> There are some typos in the new sentence you added, see suggestion.

Thanks!

**Review by junedev — APPROVED** [review]

**Pull request merged**